src/TSHelper.ts

```diff
@@ -74,7 +74,18 @@ export class TSHelper {
                 && (typeNode as ts.UnionOrIntersectionTypeNode).types.some(this.isArrayTypeNode));
     }
 
-    public static isArrayType(type: ts.Type, checker: ts.TypeChecker): boolean {
+    public static isArrayType(type: ts.Type, checker: ts.TypeChecker, dontCheckBases?: boolean): boolean {
+        if (! dontCheckBases) {
```

Comment:
Remove space between `!` and `dontCheckBases` to stay consistent with the rest of the formatting.